framework/test/unit/modules/interoperability/mainchain/commands/terminate_sidechain_for_liveness.spec.ts

```diff
@@ -100,6 +100,18 @@ describe('TerminateSidechainForLivenessCommand', () => {
 			jest.spyOn(interopMod['internalMethod'], 'isLive').mockResolvedValue(true);
 		});
 
+		it('should return error when validation fails', async () => {
+			await expect(
+				livenessTerminationCommand.verify({
+					...commandVerifyContext,
+					params: {
+						...commandVerifyContext.params,
+						chainID: Buffer.alloc(0),
+					},
+				}),
+			).rejects.toThrow("Property '.chainID' minLength not satisfied");
+		});
+
```

Comment:
instead of this we need a separate desc block "verify schema" and should have all the tests for the schema constraints